features/exploredash/src/main/res/values/strings-explore-dash.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="explore_dash">Explore Dash</string>
+    <string name="explore_subtitle">Easily shop with your DASH at over 155,000 locations and online merchants</string>
+    <string name="explore_merchants_title">Where to Spend?</string>
+    <string name="explore_merchants_subtitle">Find merchants who accept Dash as payment.</string>
+    <string name="explore_atms_title">ATMs</string>
+    <string name="explore_atms_subtitle">Find where to buy or sell DASH and other cryptocurrencies for cash.</string>
+</resources>
```

Comment:
new strings will require a change to our transifex config

make a modification to `.tx/config` to add a section for explore-dash

Here is an example from the liquid integration.  File name references are relative to the root folder of this repo:
```
[dash-mobile-wallets.strings-liquid]
file_filter = liquid-integration/src/main/res/values-<lang>/strings-liquid.xml
lang_map = sl_SI:sl, zh_TW:zh-rTW
source_file = liquid-integration/src/main/res/values/strings-liquid.xml
source_lang = en
type = ANDROID
```

Reply:
Done